### DISCUSSION TITLE
Add a Zen / Anti-distraction mode to the browser extension

### DISCUSSION THREAD

**Discussion by Programazing** [dailydotdev/daily#1465] [Feature Requests]
I love the browser extension, but sometimes my ADHD takes over, and I go on a bookmarking and reading rampage.

I want to click a button to hide the platform with a calming image, especially while working.

**Comment by nimrodkra** [comment]
@Programazing have you considered using this feature that lets you pause the new tab functionality? 
![Screenshot 2024-09-29 at 8 22 34](https://github.com/user-attachments/assets/adecc8f7-b3b5-40e5-a415-37eb34d4aa0e)
![Screenshot 2024-09-29 at 8 22 39](https://github.com/user-attachments/assets/6dfca9a5-b05b-4f58-ae84-e2f26046bbb9)

**Reply by Programazing** [reply]
I missed that. Thanks!
